src/components/Text.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export default function Text({ 
+  text = "",
+  italic = false, 
+  bold = false, 
+  underline = false, 
+  size = 'text-base', // uses tailwind utility class to set 1rem
+  color = 'text-black', 
+}) { 
+    const classNames = 
+    (italic? 'italic ' : '') + 
+    (bold? 'font-bold ' : '') + 
+    (underline? 'underline ' : '') + 
```

Comment:
(n-b) having the spacing be directly tied to the Tailwind class seems a little wonky. It can get really confusing when we start composing classNames within each other from higher level components. I'm not a Tailwind wizard so you probably know better, but I believe there may be a more elegant solution at hand.